user_docs/en/userGuide.t2t

```diff
@@ -3659,11 +3664,11 @@ Due to this, and to maintain compatibility with other screen readers in Taiwan,
 
 ++ Eurobraille displays ++[Eurobraille]
 The b.book, b.note, Esys, Esytime and Iris displays from Eurobraille are supported by NVDA.  
-These devices have a braille keyboard with 10 keys. 
+These devices have a Braille keyboard with 10 keys. 
 Of the two keys placed like a space bar, the left key is corresponding to the backspace key and the right key to the space key.
-Connected via USB, these devices have one stand-alone usb keyboard. 

```

Comment:
This sentence does not seem very clear to me nor to other translators (see [this thread](https://groups.io/g/nvda-translations/topic/line_3617_and_3619/100311411?p=,,,100,0,0,0::recentpostdate/sticky,,,100,2,0,100311411,previd%3D1690294920572050656,nextid%3D1654591085060908112&previd=1690294920572050656&nextid=1654591085060908112)). Especially with respect to the previous sentence "These devices have a Braille keyboard with 10 keys."

Could we clarify it while at it?

Reply:
@FalkoBabbage - do you have any suggestions to improve this here?

Reply:
I've opened #15194 to address this